modules/mobile/app/controllers/mobile/v0/appointments_controller.rb

```diff
@@ -137,6 +137,7 @@ def appointments_cache_interface
       def authorize
         raise_access_denied unless current_user.authorize(:vaos, :access?)
         raise_access_denied_no_icn if current_user.icn.blank?
+        raise_access_denied_no_facilities unless current_user.authorize(:vaos, :facilities?)
```

Comment:
@kpethtel , I'm unsure if this warrants a flipper. It seems simple enough but let me know if you feel differently.